docs/synthetics/ci-cd/ci-cd-installation.md

```diff
@@ -0,0 +1,283 @@
+title: CI/CD GitHub Integration Installation
+description: A guide on how to set up the CI/CD
+
+After you've created a **CI/CD Group**, you'll need a way to trigger Group Runs within your CI/CD pipelines. To accomplish this, we've created the [Sematext CI/CD GitHub Action](TODO_INSERT_OFFICIAL_GH_ACTION_LINK) which you can call within your GitHub workflows.
+
+
+## The Sematext CI/CD GitHub Action
+
+The Sematext CI/CD GitHub Action is a straightforward way to execute a **CI/CD Group Run** and collect its results. To set it up, you'll need the following:
+- A Sematext Cloud account with a Synthetics **CI/CD Group** configured
+  - You'll need the ID of your **CI/CD Group**, which you can see in the initial installation instructions when creating the group, or in the URL (for example for `https://apps.sematext.com/ui/synthetics/groups/15` the ID is `15`)
+- Your Sematext Cloud account's API key
+  - You can copy your API key from the **Settings** -> **API** page in the UI, which can be found:
+    - [here](https://apps.sematext.com/ui/account/api), if your account is registered in the US region, or
+    - [here](https://apps.eu.sematext.com/ui/account/api), for the EU region
+  - You then have to add the API key as a repository secret to the GitHub repository you wish to run these CI/CD monitors for. Go to your repository's **Settings**, and then in the sidebar under **Security** click **Secrets and variables** and then **Actions**. Create a new repository secret and name it `SEMATEXT_API_KEY`, then paste in your Sematext Cloud account's API key.
+    - If you can't see the **Settings** tab for your repository, then you might not have permissions to edit them - contact your repository's administrator to take care of this step.
+
+![CI/CD GitHub Secret](/docs/synthetics/ci-cd/images/ci-cd-github-secret.png)
+
+
+### Usage
+
+Add the following to your GitHub workflow file:
+
+```yaml
+steps:
+  - name: Run Sematext Synthetics Check
+    uses: sematext/synthetics-cicd-action@v1
+    with:
+      MONITOR_GROUP_ID: 42
+      REGION: "US"
```

Comment:
@HSSalman nitpick - single quotes to match the rest here, and how I see region specified in the YML example below